url.bs

```diff
@@ -1444,16 +1445,16 @@ actually doing that with the editors of this document first.
 <ol>
  <li><p>Let <var>output</var> be the empty string.
 
+ <li><p>Let <var>compress</var> be null.
+
  <li>
-  <p>Let <var>compress</var> be an index to the first <a>IPv6 piece</a> in the first longest
-  sequences of <var>address</var>'s <a>IPv6 pieces</a> that are 0.
+  <p>If multiple <var>address</var>'s <a for="IPv6 address">pieces</a> are 0 sequentially, then set
+  <var>compress</var> to the index of the first item of the first of the longest sequences of
+  <var>address</var>'s <a for="IPv6 address">pieces</a> that are 0.
 
   <p class=example id=example-e2b3492e>In <code>0:f:0:0:f:f:0:0</code> it would point to
   the second 0.
 
- <li><p>If there is no sequence of <var>address</var>'s <a>IPv6 pieces</a> that are 0 that is
- longer than 1, then set <var>compress</var> to null.
-
  <li><p>Let <var>ignore0</var> be false.
```

Comment:
This is probably the biggest change here.

Reply:
I think this changed from excluding 1-length sequences to including them.

FYI here is whatwg-url's implementation. It might be worth writing the whole thing out as it's pretty complex. https://github.com/jsdom/whatwg-url/blob/0fb65e3c3671c2e790ff38aed9332fc7fb2cd12b/lib/url-state-machine.js#L387

Reply:
I was hoping that multiple and sequentially would do that work here. I could rewrite this though. #321 also requested that. Hopefully next week.

Reply:
Hmm yeah, I think I skipped over the word "multiple". Sounds good to me, pending a rewrite :).